lxc/remote.go

```diff
@@ -347,9 +359,16 @@ func (c *cmdRemoteAdd) run(cmd *cobra.Command, args []string) error {
 		conf.Remotes = map[string]config.Remote{}
 	}
 
+	// Check if the first argument is a trust token. In such case, we need to
+	// decode it and use it to connect to the remote.
 	rawToken, err := shared.CertificateTokenDecode(addr)
 	if err == nil {
-		return c.runToken(server, addr, rawToken)
+		// Certificate cannot be blindly accepted when using a trust token.
+		if c.flagAcceptCert {
```

Comment:
can we re-structure this commit to avoid needing to check the same logic twice and duplicating the error message?